MIGRATION.md

```diff
@@ -13,9 +13,14 @@ When new releases include breaking changes or deprecations, this document descri
 ### Legacy API Removals
 - Dagster's legacy APIs, which were marked "legacy" in 0.13.0, have been removed. This includes `@solid`, `SolidDefinition`, `@pipeline`, `PipelineDefinition`, `@composite_solid`, `CompositeSolidDefinition`, `ModeDefinition`, `PresetDefinition`, `PartitionSetDefinition`, `InputDefinition`, `OutputDefinition`, `DynamicOutputDefinition`, `pipeline_failure_sensor`, `@hourly_schedule`, `@daily_schedule`, `@weekly_schedule`, and `@monthly_schedule`. [Here is a guide](https://docs.dagster.io/0.15.6/guides/dagster/graph_job_op) to migrating from the legacy APIs to the stable APIs.
 - Deprecated arguments to library ops have been switched to reflect stable APIs. This includes `input_defs`/`output_defs` arguments on `define_dagstermill_op`, which have been changed to `ins`/`outs` respectively, and `input_defs` argument on `create_shell_script_op`, which has been changed to `ins`.
+- The `pipeline_selection` argument has been removed from `run_failure_sensor` and related decorators / functions. Instead, use `job_selection`.
```

Comment:
job_selection will cause a deprecation warning too, the new-new arg is `monitored_jobs`

additionally, `pipeline_failure_sensor` has been removed